### PR TITLE
books: add `Conquer the Command Line`

### PR DESCRIPTION
**Pull request by CleanMachine1**
## What does this PR do?
Add resoruce

### Why is this valuable (or not)?

It provides a good set of basic information

### How do we know it's really free?

The PDF is freely available and I'm guessing that the book is creative commons.

### For book lists, is it a book? For course lists, is it a course? etc.

Book in the form of a PDF

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -373,6 +373,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Bash tutorial](https://web.archive.org/web/20180328183806/http://gdrcorelec.ups-tlse.fr/files/bash.pdf) - Anthony Scemama (PDF)
 * [BashGuide](http://mywiki.wooledge.org/BashGuide) - Maarten Billemont (HTML) [(PDF)](http://s.ntnu.no/bashguide.pdf)
 * [Conquering the Command Line](http://conqueringthecommandline.com/book/frontmatter) - Mark Bates (HTML)
+* [Essentials - Conquer the Command Line](https://magpi.raspberrypi.org/books/command-line-second-edition/pdf/download) - Richard Smedley (PDF)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:376]: ⚠️ Use the authentic book title is a must. See book cover.

**Review by davorpa — CHANGES_REQUESTED** [review]
ℹ️ Maybe changing title needs check again for list alphabetization

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (CleanMachine1) [reply books/free-programming-books-langs.md:376]: Are you referring to changing it to `Conquer the Command Line`

Comment (davorpa) [reply books/free-programming-books-langs.md:376]: You are right

Comment (CleanMachine1) [reply books/free-programming-books-langs.md]: https://github.com/EbookFoundation/free-programming-books/pull/5787/commits/3422272fa024170f4db0d6a115e8c3775d35d95b

**Review by davorpa — APPROVED** [review]
### Ready to merge :fireworks: :tada:

**Pull request merged**